e2e-cypress/Jenkinsfile.template

```diff
@@ -4,15 +4,19 @@
 
 node {
   dockerRegistry = env.DOCKER_REGISTRY
+  cypressRecordKey = env.CYPRESS_RECORD_KEY
 }
 
 odsComponentPipeline(
   podContainers: [
       containerTemplate(
         name: 'jnlp',
-        image: "${dockerRegistry}/ods/jenkins-agent-nodejs16:@shared_library_ref@",
+        image: "${dockerRegistry}/ods/jenkins-agent-nodejs18:@shared_library_ref@",
```

Comment:
The update to NodeJS 18 should be mentioned in the changelog as well.

Reply:
Seems that @roicarrera already modified the Changelog accordingly.